dsn.go

```diff
@@ -152,11 +151,21 @@ func (dsn Dsn) StoreAPIURL() *url.URL {
 	if dsn.path != "" {
 		rawURL += dsn.path
 	}
-	rawURL += fmt.Sprintf("/api/%d/store/", dsn.projectID)
+	rawURL += fmt.Sprintf("/api/%d/%v/", dsn.projectID, s)
```

Comment:
```suggestion
	rawURL += fmt.Sprintf("/api/%d/%s/", dsn.projectID, s)
```

`%s` makes the intention explicit that we want a string there, and `go vet` would catch potential type errors.